testing/matrix/linter/rules/resources/vpa.go

```diff
@@ -60,6 +60,14 @@ func isPodController(kind string) bool {
 	return kind == "Deployment" || kind == "DaemonSet" || kind == "StatefulSet"
 }
 
+func isPodControllerDaemonSet(kind string) bool {
+	return kind == "DaemonSet"
+}
+
+func isPodControllerNotDaemonSet(kind string) bool {
+	return kind == "Deployment" || kind == "StatefulSet"
```

Comment:
```suggestion
	return !isPodControllerDaemonSet(kind)
```

Reply:
Not important, i'd prefer using `!` in-place instead of introducing such a function. We replace `!` with `Not` in the name, and I cannot see a value of it.